### PR TITLE
Added mocking of metadata

### PR DESCRIPTION
**Pull request by FredrikAugust**
Metadata is now mocked!

Had to make a few changes to the model in accord with #42.

- Axes (plural) is singular
- Limit is list of numbers, not some crazy complex type (or I misunderstood BSON/MongoDB). Feel free to correct me if I missed something

### CODE REVIEW

File: API/Services/MetadataService.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        {
+            var client = new MongoClient(settings.ConnectionString);
+            var database = client.GetDatabase(settings.DatabaseName);
+
+            _storedMetadata = database.GetCollection<StoredMetadata>(settings.MetadataCollectionName);
+        }
+
+        public async Task<StoredMetadata> GetMetadata(string name)
+        {
+            Console.WriteLine(_storedMetadata.Find(new BsonDocument()).FirstOrDefault().ToString());
```
Comment (fredrikbw) [inline API/Services/MetadataService.cs:31]: Can be removed, only for testing purposes.

File: Mocks/Metadata/Metadata.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Backend.Models.Base.Metadata.POCO;
+using Bogus;
+
+namespace Backend.Mocks.Metadata
+{
+    public static class Metadata
```
Comment (fredrikbw) [inline Mocks/Metadata/Metadata.cs:8]: Can this be renamed to avoid naming conflicts? CA1724 in Sider

File: Mocks/MetAPI/Compact.cs [hunk 3]
```diff
@@ -105,7 +106,8 @@ public static Forecast GenerateSampleForecast(float lon, float lat)
                 .StrictMode(true)
                 .RuleFor(o => o.Type, f => "Feature")
```
Comment (Kpaubert) [inline Mocks/MetAPI/Compact.cs:107]: Rename f to _ as unused var

File: Mocks/MetAPI/Compact.cs [hunk 4]
```diff
@@ -105,7 +106,8 @@ public static Forecast GenerateSampleForecast(float lon, float lat)
...
                 .RuleFor(o => o.ForecastGeometry, f => forecastGeometry.Generate())
```
Comment (Kpaubert) [inline Mocks/MetAPI/Compact.cs:108]: ^

File: Mocks/MetAPI/Compact.cs [hunk 5]
```diff
@@ -105,7 +106,8 @@ public static Forecast GenerateSampleForecast(float lon, float lat)
...
-                .RuleFor(o => o.ForecastProperties, f => forecastProperties.Generate());
+                .RuleFor(o => o.ForecastProperties, f => forecastProperties.Generate())
```
Comment (Kpaubert) [inline Mocks/MetAPI/Compact.cs:109]: Etc.

**Review by Kpaubert — CHANGES_REQUESTED** [review]
Brilliant work. Just a few nitpicky thingys

**Review by Kpaubert — APPROVED** [review]
:)

**Pull request merged**